src/service/url-expander/expander.js

```diff
@@ -52,6 +65,13 @@ export class Expander {
    * @return {!Promise<string>|string}
    */
   expand(url, opt_bindings, opt_collectVars, opt_sync, opt_whiteList) {
+    this.options_ = {
```

Comment:
yep, this should go to constructor.

Reply:
Do you mean the caller? I was thinking about moving this to `url-replacements.expand...`. I don't think it could be in the `Expander` constructor since the options can change each time `expand` is called.

Reply:
Talked offline. Moved args to constructor and refactored callers to create new instance of `Expander` for each call. Done.